content/en/profiler/enabling/ddprof.md

```diff
@@ -189,7 +189,7 @@ Configuration for the profiler can be set by command line parameters, environmen
 | DD_TRACE_AGENT_URL              | url             | U          |           | `https://<hostname>:<port>` overrides other agent host/port settings.                                                                |
 | DD_TAGS                         | tags            | T          |           | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` pairs separated by commas, such as: `layer:api,team:intake`. |
 | DD_PROFILING_NATIVE_NICE        | nice            | i          |           | Sets the nice level of the profiler without affecting the instrumented processes.                                                    |
-| DD_PROFILING_NATIVE_SHOW_CONFIG | show_config     | a          | no        | Whether to log profiler configuration parameters.                                                                                    |
+| DD_PROFILING_NATIVE_SHOW_CONFIG | show_config     | c          | no        | Whether to log profiler configuration parameters.                                                                                    |
```

Comment:
```suggestion
| DD_PROFILING_NATIVE_SHOW_CONFIG | show_config     | c          | no        | Logs profiler configuration parameters.                                                                                    |
```

Reply:
Considering this is a yes / no, it is better to keep the Whether statement

Reply:
Thanks for confirming!